doc/build-osx.md

```diff
@@ -71,7 +71,8 @@ While several package managers are available for macOS, this guide will focus on
 Since the examples in this guide which walk through the installation of a package will use Homebrew, it is recommended that you install it to follow along.
 Otherwise, you can adapt the commands to your package manager of choice.
 
-To install the Homebrew package manager, see: https://brew.sh
+To install the Homebrew package manager run this command in terminal (Mac OS 11.2): /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
NACK

we don't need to post the install invocation in this doc. (https://github.com/bitcoin/bitcoin/pull/21343#discussion_r586076778)